Firebase/Auth/Source/FIRAuth_Internal.h

```diff
@@ -31,6 +31,32 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface FIRAuth ()
 
+/**
+    @brief Indicates the type of operation performed for RPCs that support the operation
+        parameter.
+ */
+typedef NS_ENUM(NSInteger, FIRAuthOperationType) {
```

Comment:
I'm more inclined to put the into its own files. FIRAuth is already too big.

Reply:
Done.